integration-tests/rita.py

```diff
@@ -312,6 +313,33 @@ def register_to_exit(node):
     os.system("ip netns exec netlab-{id} curl -XPOST 127.0.0.1:4877/settings -H 'Content-Type: application/json' -i -d '{data}'"
               .format(id=id, data=json.dumps({"exit_client": EXIT_SELECT})))
 
+def email_verif(node):
+    id = node.id
+
+    email_text = read_email(node)
+
+    code = re.search(r"\[([0-9]+)\]", email_text).group(1)
+
+    print(code)
+
+    os.system("ip netns exec netlab-{id} curl -XPOST 127.0.0.1:4877/settings -H 'Content-Type: application/json' -i -d '{data}'"
+              .format(id=id, data=json.dumps({"exit_client": {"exits": {"exit_a": {"email_code": code}}}})))
+
+    os.system("ip netns exec netlab-{id} curl 127.0.0.1:4877/settings"
+              .format(id=id))
+
+
+
+def read_email(node):
+    id = node.id
+    # TODO: this is O(n^2)
+    for mail in os.listdir("mail"):
+        with open(os.path.join("mail", mail)) as mail_text:
+            mail = json.load(mail_text)
+            if mail["envelope"]["forward_path"][0] == "{}@althea.com".format(id):
```

Comment:
`althea.com` is an existing domain name, apparently with a mailing system set up. I don't think we want to necessarily ever risk spamming someone's server. `example.com` OTOH is specifically designated not to be used for anything other than testing.

Reply:
ok, will change